Carnap-Server/Handler/Instructor.hs

```diff
@@ -90,11 +86,12 @@ putInstructorR ident = do
 
 deleteInstructorR :: Text -> Handler Value
 deleteInstructorR ident = do
-    msg <- requireJsonBody :: Handler InstructorDelete
+    msg <- requireCheckJsonBody :: Handler InstructorDelete
     case msg of
-      DeleteAssignment id ->
-        do datadir <- appDataRoot <$> (appSettings <$> getYesod)
-           deleted <- runDB $ deleteCascade id
+      DeleteAssignment aid ->
+        -- XXX: forgot to delete the file?
```

Comment:
Is this a bug that we're not deleting the assignment? I saw the code was redundant but wasn't sure if it was half removed or half added.

Reply:
Not a bug, but yeah, some sloppiness, I don't think there's a need to get the dataroot  here. Deleting an assignment should NOT delete the underlying document, since the instructor might want to use that document again, or might be using it in another class etc.